src/cache/memo-cache.utils.ts

```diff
@@ -23,16 +27,57 @@ export function memoCacheKey<F extends MemoizableFunction>(f: F, { resolver, ...
     const key = resolverFunction(...args);
     const cache = memoized.cache;
 
-    if (cache.has(key)) return cache.get(key);
+    if (cache.has(key)) return cache.getValue(key);
 
     const result = f(...args);
 
-    cache.set(key, result);
+    cache.setValue(key, result);
 
     return result;
   };
 
+  const skipCache: (...args: Parameters<F>) => ReturnType<F> = (...args) => {
+    return f(...args);
+  };
+
+  const refresh: (...args: Parameters<F>) => ReturnType<F> = (...args) => {
+    const key = resolverFunction(...args);
```

Comment:
Missing:
- The default resolver function might generate super long keys (e.g. if one of the params is a text blob). It would make sense to hash that (maybe only in some cases).

Reply:
I also need to DRY this logic here, I just duplicated the same functions a few times with some changes.